src/api/BundlePush/BundlePusher.ts

```diff
@@ -644,24 +644,58 @@ export class BundlePusher {
   private async gatherNodejsDiagnosticsFromCics(cicsSession: AbstractSession): Promise<boolean> {
```

Comment:
This method is getting quite long and does a couple of different things, suggest breaking out the code for querying the Nodejsapps and getting the Regions into separate methods.